client/components/language-picker/modal.jsx

```diff
@@ -580,7 +604,7 @@ export class LanguagePickerModal extends PureComponent {
 			<Dialog
 				isVisible
 				buttons={ buttons }
-				onClose={ this.handleClose }
+				onClose={ () => this.handleClose( true ) }
```

Comment:
Ideally, we should have a consistent way to add those handlers. Right now we both add `handleClose` directly and in a wrapper function.